### ISSUE TITLE
Forward selection is not working

### ISSUE THREAD

**Issue by timcdlucas** [timcdlucas/INLAutils#40]
```
  # Try and make a dataset where a variable WILL get added.
  Epil2 <- Epil
  Epil2$Base <- Epil$y + rnorm(nrow(Epil), sd = 0.01)
  Epil2$Age <- Epil$y + rnorm(nrow(Epil), sd = 0.01)
  
  stack2 <- inla.stack(data = list(y = Epil2$y),
                      A = list(1),
                      effects = list(data.frame(Intercept = 1, Epil2[3:5])))
  
  result1 <- INLAstep(fam1 = "poisson", 
                      Epil2,
                      in_stack = stack2,
                      invariant = "0 + Intercept",
                      direction = 'forwads',
                      include = 3:5,
                      y = 'y',
                      y2 = 'y',
                      powerl = 1,
                      inter = 1,
                      thresh = 0.001)
  ```

**Comment by timcdlucas** [comment]
Forwards is spelt wrong you idiot.

**Comment by timcdlucas** [comment]
Fixed by e66513337bccaf5c03d748323b5223885c3cd294
